src/_posts/platform/app/task-scheduling/2000-01-01-scalingo-scheduler.md

```diff
@@ -1,42 +1,132 @@
 ---
 title: Scalingo Scheduler - Run Scheduled Tasks
 nav: Scalingo Scheduler
-modified_at: 2022-10-24 10:00:00
+modified_at: 2023-01-31 17:00:00
 tags: task-scheduling
 index: 1
 ---
 
-Scalingo Scheduler is here to help you run scheduled tasks at regular intervals.
+Scalingo provides a feature called *Scalingo Scheduler* to help you run tasks
+on a regular basis.
 
-The syntax used to describe this interval is the same as in the cron software.
+## About the Scalingo Scheduler
 
-Scalingo Scheduler launches tasks as [one-off containers]({% post_url platform/app/2000-01-01-tasks %}) in detached mode. Therefore the related one-off documentation and their detached mode applies.
+As the name suggests, the Scalingo Scheduler allows to define tasks so that
+they run periodically at fixed times, dates, or intervals. It can be leveraged
+to do a variety of tasks such as sending a newsletter every morning, exporting
+data every hour, checking for new orders every 15 minutes, ...
 
-{% warning %}
-Scheduled tasks execution is expected but not guaranteed. Scalingo Scheduler is known to occasionally (but rarely) miss the execution of scheduled jobs. If scheduled tasks are a critical component of your application, it is recommended to [run a custom clock process]({% post_url platform/app/task-scheduling/2000-01-01-custom-clock-processes %}) instead for more reliability, control, and visibility.
-{% endwarning %}
+The Scalingo Scheduler launches tasks in [one-off containers]({% post_url platform/app/2000-01-01-tasks %})
+running in detached mode. Therefore, [the related one-off documentation]({% post_url platform/app/2000-01-01-tasks %})
+fully applies.
 
-## Defining Tasks
+The Scalingo Scheduler also works with [Review Apps]({% post_url platform/app/2000-01-01-review-apps %}),
+just like the parent application does.
 
-Scheduled tasks are defined by adding a `cron.json` file at the root of your application's source code.
+### Limitations
```

Comment:
suggestion: I would move this section after explaining how to use the feature. It feels weird to mention the limitations _before_ explaining the feature in itself.

Reply:
I made this on purpose :D :

While doing my support shift, I've seen (twice) people implementing it and then realizing there were limitations. I'd like to avoid this kind of situations, because I think they create a lot of frustrations.

So, the goal here is to have the limitations explained quite early, so people can see if this feature will fit their needs. If it is, they will read along for implementation instructions. If it's not, they will switch to another solution without having to read all the (useless, in this case) implementation details.

The goal is really to allow users decide if this will fit their needs as soon as possible, so they don't lose time reading/doing stuff and then discovering that it won't work.